assets/scss/components/_reminders.scss

```diff
@@ -0,0 +1,130 @@
+/*
+Reminder List
+
+Display an Unordered List of reminder items with time formatted in a new line.
+
+Markup:
+<ul class="reminder-list">
+  <li>Reminder List Item 1</li>
+  <li>Reminder List Item 2</li>
+  <li>Reminder List Item 3 <time>Reminder Time</time></li>
+</ul>
+
+
+Styleguide Reminders.reminder-list
```

Comment:
I'm sure I commented on this last week needs to be `Reminders.List` rather than `Reminders.reminder-list` remember you are already in `Reminders`